adal/src/main/java/com/microsoft/aad/adal/AuthenticationActivity.java

```diff
@@ -726,7 +726,7 @@ public void alias(String alias) {
     /**
      * handle spinner display.
      * 
-     * @param show
+     * @param show boolean flag, pass true if spinner needs to be displayed
```

Comment:
Usually the way to javadoc boolean variable is @param show True if spinner needs to be displayed, false otherwise.